railgun/manager/manager.go

```diff
@@ -41,10 +44,23 @@ type Config struct {
 	KubeconfigPath       string
 
 	ZapOptions zap.Options
+
```

Comment:
afaik these fields are only going to exist temporarily to get us to our first alpha release faster but we expect to remove them and provide backwards compat with 1.x before beta releases start. As such, we should provide a comment here explaining that context.